src/runtime_src/core/tools/common/ReportCmcStatus.cpp

```diff
@@ -78,12 +83,18 @@ ReportCmcStatus::getPropertyTree20202( const xrt_core::device * _pDevice,
       pwr_override_data.put("power_watts", pt.pwr_scaling_ovrd_limit);
       runtime_tree.add_child("power_threshold_override", pwr_override_data);
 
-      cmc_tree.add_child("scaling", runtime_tree);
+      cmc_tree.add_child("throttling", runtime_tree);
     }
   }
   catch(const xrt_core::query::no_such_key&) {}
   catch(const xrt_core::query::sysfs_error&) {}
 
+  xrt::device device(_pDevice->get_device_id());
+  std::stringstream ss;
+  ss << device.get_info<xrt::info::device::dynamic_regions>();
+  boost::property_tree::read_json(ss, device_stat_tree);
+  cmc_tree.add_child("device_stats", device_stat_tree);
```

Comment:
We definitely should not be placing the entire dynamic region report into the cmc property tree.

If we want the xclbin uuid there are better ways of getting it other than going the dynamic region report.
```suggestion
  cmc_tree.add_child("xclbin_uuid", _pDevice->get_xclbin_uuid());
```
We could also just state if the xclbin is valid through the property tree
```suggestion
  cmc_tree.add_child("cmc_status_valid", boost::iequals(_pDevice->get_xclbin_uuid(), N/A"));
```